.github/workflows/fuzz-test.yml

```diff
@@ -0,0 +1,55 @@
+name: Execute Fuzzing Tests
+
+on:
+  push:
+    branches:
+      - "master"
+      - "*-rc"
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: 'Branch for Fuzzing Tests'
+        required: false
+        default: 'master'
+
+jobs:
+  fuzz-test:
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Validate branch name
+        id: validate-branch
+        env:
+            GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+            github_event_inputs_branch: ${{ github.event.inputs.branch }}
+            github_ref_name: ${{ github.ref_name }}
+        run: |
+          
+          if [[ ! "${{ github_event_inputs_branch || github_ref_name }}" =~ ^[a-zA-Z0-9._/-]+$ ]]; then
+            echo "Invalid branch name: ${{ github_event_inputs_branch || github_ref_name }}"
+            exit 1
+          fi        
+
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 #v4.2.2
+        with:
+          ref: ${{ github.event.inputs.branch || github.ref_name }}
+
+      - name: Setup Java & Gradle
+        uses: actions/setup-java@3a4f6e1af504cf6a31855fa899c6aa5355ba6c12 #v4.7.0
+        with:
+          java-version: '17'
+          distribution: 'temurin'
+          cache: 'gradle'
+
+      - uses: actions/cache/restore@d4323d4df104b026a6aa633fdb11d772146be0bf #v4.2.2
+        name: Restore Gradle Wrapper
+        with:
+          path: |
+            gradle/wrapper/gradle-wrapper.jar
+          key: gradle-wrapper-v1
+          fail-on-cache-miss: true
+
+      - name: Run fuzz tests
+        run: |
+          #Adding fuzzTest instead of runAllFuzzTests to avoid running all the fuzz tests but it is something to evaluate. 
+          ./gradlew --no-daemon --stacktrace fuzzTest
```

Comment:
This would be the right command for doing the full test `JAZZER_FUZZ=1 ./gradlew runAllFuzzTests --info --continue`